src/Http/Http.Abstractions/src/Internal/ParsingHelpers.cs

```diff
@@ -130,4 +135,12 @@ public static void AppendHeaderUnmodified(IHeaderDictionary headers, string key,
         var existing = GetHeaderUnmodified(headers, key);
         SetHeaderUnmodified(headers, key, StringValues.Concat(existing, values));
     }
+
+    [InlineArray(4)]
```

Comment:
I’ve been using the collection initializer trick recently:

    Span<string> stackBuffer = [null, null, null, null];

Reply:
Yup, that's possible, just at the moment it leads to fairly ugly IL:
[SharpLab](https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoBAGABAlAOgCUBXGAFyjAFNCBhAezAAcoAbGgJwGVuA3KAGMaAZwDcmTAgDM+JLnq4A3plzr8c2BVwBZFAAoAlGo2qMGy/gDsuAGKNGBgNoxS7dgBpcbj999ePu7sALpGkhYaAL6m6rGauNp6SMbx5lYaDqS8FFywAOaiuMCkAGal3LgAvLgAJjSlEO4UERnqCLYOTiXl3OHxMZFxQ7jOujQUABaMtQCSrOwG41Mz8yzsAPIsVIwwooQAcoyzMOywBUYh8Sx5/BAUNImU9o4GPCwQMAA8BNgAfLhRB8YEZqgCACxIVoaeLOE5nGA0ACCXC4EAAngZwZdrrd7o9RLlSEIdFkcnkYIU0vFLDcoHcHvgUHgAPp3dikGjQ9SDKJAA)
The JIT does a good job of making them the same, though.